Cargo.toml

```diff
@@ -41,7 +41,7 @@ chacha20-poly1305-aead = { version = "^0.1", optional = true }
 blake2-rfc = { version = "^0.2", optional = true }
 rust-crypto = { version = "^0.2", optional = true }
 hacl-star = { version = "=0.0.11", optional = true }
-rand = { version = "0.5.0-pre.2", optional = true }
+rand = { version = "0.5.4", optional = true }
```

Comment:
Personally, I prefer explicit semver range specifiers, and prefer to reduce specificity where possible and when it's not a security/functionality concern, so: `^0.5` is better for me.

If another crate is stuck on `0.5.3` for some reason and `snow` is compatible with it, I don't see why we need to hold them back.